pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.532</version>
+        <version>1.598-SNAPSHOT</version> <!-- TODO SimpleBuildWrapper-JENKINS-24673 -->
```

Comment:
Depends on https://github.com/jenkinsci/jenkins/pull/1530 and (to be useful) https://github.com/jenkinsci/workflow-plugin/pull/37.